### PR TITLE
docs: requeue

### PR DESCRIPTION
**Pull request by cmeesters**
only documenting what is new in v0.11.0 (see #152 )

## Summary by CodeRabbit

- **New Features**
	- Introduced new command line flags for the SLURM executor plugin: `--slurm_init_seconds_before_status_checks` and `--slurm_requeue`.
- **Documentation**
	- Expanded guidance on job resubmission and retries, enhancing clarity on the use of `--retries` and `--slurm-requeue`.
	- Improved overall structure of the documentation for better understanding of job configuration and resource specifications.
	- Minor formatting corrections made for clarity.

### CODE REVIEW

**Comment by cmeesters** [comment]
Would be nice if one of you would find the time to review. Thank you!

File: docs/further.md [hunk 1]
```diff
@@ -160,6 +160,15 @@ set-resources:
         cpus_per_task: 40
 ```
 
+### Additional Command Line Flags
+
+This plugin defines additional command line flags. As always the can be used on the command line or in a profile.
```
Comment (dlaehnemann) [inline docs/further.md:165]: Mini rephrasing of the second sentence. And I prefer having one sentence per line in Markdown files, for clearer git diffs down the line...
```suggestion
This plugin defines additional command line flags.
As always, these can be set on the command line or in a profile.
```

File: docs/further.md [hunk 2]
```diff
@@ -160,6 +160,15 @@ set-resources:
...
+
+| Flag        | Meaning  |
+|-------------|----------|
+| `--slurm_init_seconds_before_status_checks`| will modify the default time (40 seconds) before the initial status check - usefull for development purposes|
```
Comment (dlaehnemann) [inline docs/further.md:169]: Just expanding a bit on the reasoning behind the default and where shorter wait times might be desirable.
```suggestion
| `--slurm_init_seconds_before_status_checks`| modify time before initial job status check; the default of 40 seconds avoids load on querying slurm databases, but shorter wait times are for example useful during workflow development |
```

File: docs/further.md [hunk 3]
```diff
@@ -271,7 +280,7 @@ export SNAKEMAKE_PROFILE="$HOME/.config/snakemake"
 
 ==This is ongoing development. Eventually you will be able to annotate different file access patterns.==
 
-## Retries - Or Trying again when a Job failed
+## <a name="retries"></a> Retries - Or Trying again when a Job failed
```
Comment (dlaehnemann) [inline docs/further.md:283]: Are you sure, this manual html link is necessary? Usually, you should be able to link within a markdown document, directly referring to headings (with whitespace replaced by dashes `-`):
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown

File: docs/further.md [hunk 4]
```diff
@@ -282,7 +291,21 @@ snakemake --retries=3
...
+The "requeue" option allows jobs to be resubmitted automatically if they fail or are preempted. This might be the default on your cluster, already. You can check your cluster's requeue settings with 
+
+```console
+scontrol show config | grep Requeue
+```
+
+This requeue feature is integrated into the SLURM submission command, adding the --requeue parameter to allow requeuing after node failure or preemption using:
+
+```console
+snakemake --slurm-requeue ...
```
Comment (dlaehnemann) [inline docs/further.md:308]: How is this different from snakemake's own `--retries` option? Or does this simply implement the `--retries` value for pre-empted jobs or does it simply move the retries to the level of the slurm scheduler? Maybe you can clarify this further?

**Review by dlaehnemann — CHANGES_REQUESTED** [review]
Nice extra features, and great to have them documented. I have some questions and suggestions.

File: docs/further.md (hunk 2 above)
Comment (cmeesters) [reply docs/further.md]: thank you

File: docs/further.md (hunk 4 above)
Comment (cmeesters) [reply docs/further.md]: Right. This requires some SLURM background. I tried to explain in a single sentence in commit cb153d2e21f65def715c26f728e9cd98f9be4188 .

File: docs/further.md (hunk 3 above)
Comment (cmeesters) [reply docs/further.md]: let's see. The html is removed in ed0c0d0b02ba902504b5b0fb97bd6fb4c2e6c32f

**Review by dlaehnemann — APPROVED** [review]
Thanks for the updates!

**Comment by cmeesters** [comment]
> Thanks for the updates!

Thank you for your prompt review and the lectoring!

**Pull request merged**